keras/utils/vis_utils.py

```diff
@@ -18,7 +18,10 @@ def _check_pydot():
                           ' and graphviz for `pydotprint` to work.')
 
 
-def model_to_dot(model, show_shapes=False, show_layer_names=True):
+def model_to_dot(model,
+                 show_shapes=False,
+                 show_layer_names=True,
+                 rankdir='TB'):
     """Converts a Keras model to dot format.
 
     # Arguments
```

Comment:
Ideally we should write a test to automatically check docstring existence and correctness. This is on our to-do list: https://github.com/fchollet/keras/projects/1